### PR TITLE
chore(localdebug): allow deleting dev tunnels if tunnel number exceed the limitation

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Detail: [Feature 17793817](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17793817): Allow deleting dev tunnels if tunnel number exceed the limitation

Information dialog:
image

Output:
image

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"It seems that the tunnel in your account has exceeded its limit. Would you like to delete all tunnels created through Teams Toolkit?",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelListMessage":"The following is the dev tunnels created by Teams Toolkit in your account:",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceeded.yes": "Yes",
```
Comment (MuyangAmigo) [inline packages/vscode-extension/package.nls.json:298]: Can we change `Yes` to a clear Call to Action: `Delete all tunnels`

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -292,6 +292,11 @@
...
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceeded.no": "No",
```
Comment (MuyangAmigo) [inline packages/vscode-extension/package.nls.json:299]: Can we change `No` to `Cancel`.

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"It seems that the tunnel in your account has exceeded its limit. Would you like to delete all tunnels created through Teams Toolkit?",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelListMessage":"The following is the dev tunnels created by Teams Toolkit in your account:",
```
Comment (MuyangAmigo) [inline packages/vscode-extension/package.nls.json:297]: You have reached maximum numbers of tunnel created with your Microsoft 365 account. Clean up tunnels to continue debugging your Teams application. Refer to Output channel to review tunnel details.

**Review by MuyangAmigo — DISMISSED** [review]

**Review by swatDong — DISMISSED** [review]

**Review by kuojianlu — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json [hunk 4]
```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"To continue debugging your Teams application, ensure all debugging Teams applications are closed and clean up the dev tunnels that were not properly removed by Teams Toolkit. Refer to [output channel](%s) to review tunnel details.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelListMessage":"You have reached maximum numbers of tunnel created with your Microsoft 365 account. The following is the dev tunnels created by Teams Toolkit in your account:",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:297]: numbers of tunnel -> number of tunnels
The following is the dev tunnels created by Teams Toolkit in your account: -> The dev tunnels in your account:

File: packages/vscode-extension/package.nls.json [hunk 5]
```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:295]: Does "Dev tunnel"'s 'D' need to be capitalized here?

File: packages/vscode-extension/package.nls.json [hunk 6]
```diff
@@ -292,6 +292,11 @@
...
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"To continue debugging your Teams application, ensure all debugging Teams applications are closed and clean up the dev tunnels that were not properly removed by Teams Toolkit. Refer to [output channel](%s) to review tunnel details.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:296]: Exceeded dev tunnel limit. Clean up unused tunnels, close other debugging sessions and retry. Check [output channel](%s) for more details.

File: packages/vscode-extension/package.nls.json (hunk 4 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

File: packages/vscode-extension/package.nls.json (hunk 5 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

File: packages/vscode-extension/package.nls.json (hunk 6 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

**Review by sffamily — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Review by a1exwang — APPROVED** [review]

**Pull request merged**